boards.txt

```diff
@@ -35828,4 +36000,188 @@ Geekble_ESP32C3.menu.EraseFlash.none.upload.erase_cmd=
+esp32c6.menu.FlashMode.dio.build.boot=dio
+
+esp32c6.menu.FlashFreq.80=80MHz
+esp32c6.menu.FlashFreq.80.build.flash_freq=80m
+esp32c6.menu.FlashFreq.40=40MHz
+esp32c6.menu.FlashFreq.40.build.flash_freq=40m
+
+esp32c6.menu.FlashSize.4M=4MB (32Mb)
+esp32c6.menu.FlashSize.4M.build.flash_size=4MB
+esp32c6.menu.FlashSize.8M=8MB (64Mb)
+esp32c6.menu.FlashSize.8M.build.flash_size=8MB
+esp32c6.menu.FlashSize.8M.build.partitions=default_8MB
+esp32c6.menu.FlashSize.2M=2MB (16Mb)
+esp32c6.menu.FlashSize.2M.build.flash_size=2MB
+esp32c6.menu.FlashSize.2M.build.partitions=minimal
+esp32c6.menu.FlashSize.16M=16MB (128Mb)
+esp32c6.menu.FlashSize.16M.build.flash_size=16MB
+
+esp32c6.menu.UploadSpeed.921600=921600
+esp32c6.menu.UploadSpeed.921600.upload.speed=921600
+esp32c6.menu.UploadSpeed.115200=115200
+esp32c6.menu.UploadSpeed.115200.upload.speed=115200
+esp32c6.menu.UploadSpeed.256000.windows=256000
+esp32c6.menu.UploadSpeed.256000.upload.speed=256000
+esp32c6.menu.UploadSpeed.230400.windows.upload.speed=256000
+esp32c6.menu.UploadSpeed.230400=230400
+esp32c6.menu.UploadSpeed.230400.upload.speed=230400
+esp32c6.menu.UploadSpeed.460800.linux=460800
+esp32c6.menu.UploadSpeed.460800.macosx=460800
+esp32c6.menu.UploadSpeed.460800.upload.speed=460800
+esp32c6.menu.UploadSpeed.512000.windows=512000
+esp32c6.menu.UploadSpeed.512000.upload.speed=512000
+
+esp32c6.menu.DebugLevel.none=None
+esp32c6.menu.DebugLevel.none.build.code_debug=0
+esp32c6.menu.DebugLevel.error=Error
+esp32c6.menu.DebugLevel.error.build.code_debug=1
+esp32c6.menu.DebugLevel.warn=Warn
+esp32c6.menu.DebugLevel.warn.build.code_debug=2
+esp32c6.menu.DebugLevel.info=Info
+esp32c6.menu.DebugLevel.info.build.code_debug=3
+esp32c6.menu.DebugLevel.debug=Debug
+esp32c6.menu.DebugLevel.debug.build.code_debug=4
+esp32c6.menu.DebugLevel.verbose=Verbose
+esp32c6.menu.DebugLevel.verbose.build.code_debug=5
+
+esp32c6.menu.EraseFlash.none=Disabled
+esp32c6.menu.EraseFlash.none.upload.erase_cmd=
+esp32c6.menu.EraseFlash.all=Enabled
+esp32c6.menu.EraseFlash.all.upload.erase_cmd=-e
+
+esp32c6.menu.ZigbeeMode.default=Disabled
+esp32c6.menu.ZigbeeMode.default.build.zigbee_mode=
+esp32c6.menu.ZigbeeMode.default.build.zigbee_libs=
+esp32c6.menu.ZigbeeMode.ed=Zigbee ED (end device)
+esp32c6.menu.ZigbeeMode.ed.build.zigbee_mode=-DZIGBEE_MODE_ED
+esp32c6.menu.ZigbeeMode.ed.build.zigbee_libs=-lesp_zb_api_ed -lesp_zb_cli_command -lzboss_stack.ed.trace -lzboss_stack.ed -lzboss_port
+esp32c6.menu.ZigbeeMode.zczr=Zigbee ZCZR (coordinator)
+esp32c6.menu.ZigbeeMode.zczr.build.zigbee_mode=-DZIGBEE_MODE_ZCZR
+esp32c6.menu.ZigbeeMode.zczr.build.zigbee_libs=-lesp_zb_api_zczr -lesp_zb_cli_command -lzboss_stack.zczr.trace -lzboss_stack.zczr -lzboss_port
+esp32c6.menu.ZigbeeMode.rcp=Zigbee RCP (radio co-processor)
+esp32c6.menu.ZigbeeMode.rcp.build.zigbee_mode=-DZIGBEE_MODE_RCP
+esp32c6.menu.ZigbeeMode.rcp.build.zigbee_libs=-lesp_zb_api_rcp -lesp_zb_cli_command -lzboss_stack.rcp -lzboss_port
```

Comment:
Please remove this esp32c6 as we already have the definition for it. I assume its just left over when you were adding your board.